summary/summary.go

```diff
@@ -136,9 +136,15 @@ func Summary(c *k8s.Client, o Options) error {
 	if err != nil {
 		return err
 	}
-	for _, sum := range summary {
+	for i, sum := range summary {
 		if o.Output == "json" {
-			fmt.Printf("%s", sum)
+			if i == 0 {
+				fmt.Printf("[%s,\n", sum)
+			} else if i == len(summary)-1 {
+				fmt.Printf("%s]\n", sum)
+			} else {
```

Comment:
We shouldn't use this way to manually modify the JSON string. Let's push the json objects to a array and then marshal it. That would marshal and output it in the correct format.

Reply:
OK